core/primitives-core/src/account.rs

```diff
@@ -231,12 +343,20 @@ impl serde::Serialize for Account {
         S: serde::Serializer,
     {
         let version = self.version();
+        let code_hash = match self.contract() {
```

Comment:
would be nice to have that as part of `Account` api for consistency with `global_contract_hash` and `global_contract_account_id`: `fn local_contract_hash(&self) -> Option<CryptoHash>` and then do `code_hash = self.local_contract_hash().unwrap_or_default()`
also replace all usages of `account.contract().local_code()` with just `account.local_contract_hash()`, we have quite a few of though throughout the codebase